client/landing/gutenboarding/components/confirm-purchase-modal/style.scss

```diff
@@ -0,0 +1,37 @@
+.confirm-purchase-modal__content {
+	padding: 32px;
+	max-width: 560px;
+}
+
+.confirm-purchase-modal__header {
+	font-family: 'Recoleta-Regular', 'Recoleta', serif;
```

Comment:
We don't actually have this font defined yet anywhere. It's defined at this unmerged PR:

https://github.com/Automattic/wp-calypso/pull/39928/files#diff-afb627d6ab2284b3649ebea7dc364108R1

I'd suggest either bringing over those font definitions from there or drop them out here and add back later.